improver/nbhood/recursive_filter.py

```diff
@@ -336,69 +284,97 @@ def _run_recursion(
             cube.data = output
         return cube
 
-    def _set_smoothing_coefficients(
-        self, cube, smoothing_coefficient, smoothing_coefficients_cube
-    ):
-        """
-        Set up the smoothing_coefficient parameter.
+    def _validate_smoothing_coefficients(self, cube, smoothing_coefficients_cube):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5583761)